### PR TITLE
Profile ViewModel Unit Test

### LINKED ISSUE #60: Unit Testing for Profile ViewModel

### PR DESCRIPTION
**Pull request by cyrusdiego**
# Summary
- Adds unit testing for Profile ViewModel
- Updated Gradle 

Contributes to #53 
Closes #60

### CODE REVIEW

File: app/src/test/java/com/example/unlibrary/profile/ProfileViewModelTest.java [hunk 1]
```diff
@@ -1,34 +1,141 @@
...
+
+        mMockObserver = (Observer<Pair<AuthUtil.InputKey, String>>) mock(Observer.class);
+        mViewModel.getInvalidInputEvent().observeForever(mMockObserver);
+    }
 
     @Test
-    public void test() {
-        assertEquals(1, 1);
+    public void testGetUser() {
+        assertEquals(mUser, getOrAwaitValue(mViewModel.getUser()));
```
Comment (calebschoepp) [inline app/src/test/java/com/example/unlibrary/profile/ProfileViewModelTest.java:101]: How come user starts initiated but password does not?

**Review by calebschoepp — APPROVED** [review]
Overall LGTM. Any reason in particular gradle needed to be updated?

File: app/src/test/java/com/example/unlibrary/profile/ProfileViewModelTest.java (hunk 1 above)
Comment (cyrusdiego) [reply app/src/test/java/com/example/unlibrary/profile/ProfileViewModelTest.java:101]: the password is only set when we are editing the user information, when they type it in to re-authenticate so initially it is an empty string. Also regarding the gradle update, no particular reason, just saw that it wasn't up to date and thought we can upgrade it

Comment (calebschoepp) [reply app/src/test/java/com/example/unlibrary/profile/ProfileViewModelTest.java:101]: Okay, I'm happy with those answers.

**Pull request merged**